.storybook/3.COLORS.stories.mdx

```diff
@@ -18,7 +18,7 @@ We follow a 3 tiered system for color design tokens and css variables.
 <div
   style={{
     textAlign: 'center',
-    backgroundColor: 'var(--brand-colors-white-white000)',
+    backgroundColor: 'var(--brand-colors-white)',
```

Comment:
### After

Background color to image still works as expected


https://github.com/MetaMask/metamask-extension/assets/8112138/5d112637-07db-40ac-95ae-575691e40209